src/code_F5750.c

```diff
@@ -142,7 +167,47 @@ void PlayerJump2(ScriptInstance* script, s32 isInitialCall) {
     player_jump(script, isInitialCall, 2);
 }
 
-INCLUDE_ASM(s32, "code_F5750", InterpPlayerYaw, ScriptInstance* script, s32 isInitialCall);
+ApiStatus InterpPlayerYaw(ScriptInstance* script, s32 isInitialCall) {
+    Bytecode* args = script->ptrReadPos;
+    PlayerStatus* playerStatus = PLAYER_STATUS;
+    f32* t1 = &script->functionTemp[1].f;
+    f32* t2 = &script->functionTemp[2].f;
+    s32* t3 = &script->functionTemp[3].s;
```

Comment:
Would be cool to give these clear names.... I'm thinking `initialYaw`, `deltaYaw`, `time` respectively.